CHANGELOG.md

```diff
@@ -7,6 +7,8 @@
 -   Added `chevronColor` property onto `<InfoListItem>`.
 -   Added `chevronIcon` class for `chevron` element in `<InfoListItem>`.
 -   Added 1rem default padding to the root styles of `<EmptyState>`. ([#320](https://github.com/brightlayer-ui/react-component-library/issues/320))
+-   Added new property `unitSpace` to `<ChannelValue>` to manage spacing between the value and units. ([#350](https://github.com/brightlayer-ui/react-component-library/issues/350))
+-   In `<Hero>` component, `fontSize`, `value`, `valueIcon`, `valueColor` and `units` props have been replaced by `ChannelValueProps` prop, which will allow you to specify any props on the underlying `<ChannelValue>` component.
```

Comment:
If it is "replaced", it would be a breaking change, and should not be published in this round.

Reply:
Sure I have created an feature request here https://github.com/brightlayer-ui/react-component-library/issues/365 (for major release). I will remove above changes and update PR.

Reply:
We can release this update now if we have both the individual channel value props(value, units, unitSpace... with a deprecation warning), along with the updated ChannelValueProps. Then in the major version, we will remove the deprecation warning and extra props.

Reply:
Yesterday confirmed with Joe to create a new feature request https://github.com/brightlayer-ui/react-component-library/issues/365 (for major release).